docs/components/LessStyleGuideView.jsx

```diff
@@ -1,14 +1,233 @@
 import React from "react";
 
+import Example from "./Example";
 import View from "./View";
+import {Grid} from "src";
 
 
 export default function LessStyleGuideView() {
   const {cssClass} = LessStyleGuideView;
+  const {Col, Row} = Grid;
 
   return (
     <View className={cssClass.CONTAINER} title="Less Style Guide">
-      TODO
+      <Grid>
+        <Row grow className="margin--bottom--xl">
+          <Col span={6} className="self--start padding--right--l">
+            <h2>Naming Conventions</h2>
+            <p>Dewey uses a mix of BEM and Atomic CSS for naming conventions. It is influenced by&nbsp;
+            <a href="http://basscss.com/">Basscss</a> for utility classes, while components are
+            written with a BEM notation.</p>
+
+            <Example
+              title="Variables"
+              code={`
+                  @zIndex--1: 100;
+              `}
+            >
+              Variables are the most atomic building blocks of Dewey. Each variable defines a single value for a
+              specific attribute. Variables are lower-cased camelCase with a double dash <code>--</code> for modifiers.
```

Comment:
nit: can replace `lower-cased camelCase` with `camelCase` and `title-cased CamelCase` with `PascalCase` below